README.md

```diff
@@ -20,7 +20,7 @@ Right now, it powers:
 use taffy::prelude::*;
 
 // First create an instance of Taffy
-let mut taffy = Taffy::new();
+let mut taffy: Taffy<MeasureFunc<()>> = Taffy::new();
```

Comment:
I can try it, but I'm guessing not on the basis that `Measure` defaults to `MeasureFunc<()>` so if that were possible then it ought to be possible to just write `Taffy` (but I tried that and it didn't work). I'm thinking maybe an alias like `type DefaultTree = Taffy<MeasureFunc<()>>`?

See also: https://github.com/DioxusLabs/taffy/issues/449#issuecomment-1572935359

Reply:
My latest thinking on this is that we ought to rename the generic struct to `TaffyTree`, and then have a `type Taffy = TaffyTree<MeasureFunc<()>>` for backwards compatibility, but otherwise encourage users of Taffy to define their own alias, and documenting a bunch of helpful examples on the `TaffyTree` struct.